lib/src/app/app.dart

```diff
@@ -18,6 +18,15 @@ class AppClient {
 
   AppClient(this._client);
 
+  /// List LocationSummary objects which contain machine summaries within them for an organization
```

Comment:
Hmm... sorry for the back and forth on this! I think the important part of this is the machine summaries (`listLocations` exists if someone wants location data specifically), but we want to address the fact that they're returned as `LocationSummary`s. Maybe something like `/// List all [MachineSummary]s, grouped by location.`?